circuitbreaker/hystrix.go

```diff
@@ -0,0 +1,55 @@
+package circuitbreaker
+
+import (
+	"errors"
+
+	"github.com/afex/hystrix-go/hystrix"
+	"github.com/go-kit/kit/endpoint"
```

Comment:
goimports style: move this import statement to a third group, separated by a blank line, please :)